tests/functional/spec/list_collector_section_summary_items.spec.js

```diff
@@ -0,0 +1,139 @@
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      expect(browser.getUrl()).to.contain(AnyCompaniesOrBranchesPage.url());
+    });
+    it("When I add my own item and relevant data, Then after submitting I should be asked if I have more items to add.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      expect($(AnyCompaniesOrBranchesPage.yes()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesPage.no()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesPage.heading()).getText()).to.contain(
+        "Do you need to add any other UK companies or branches that undertake general insurance business?"
+      );
+    });
+    it("When I add my own item and relevant data, Then after I answer no on additional items page I should get to the section summary page.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      $(AnyCompaniesOrBranchesPage.no()).click();
+      $(AnyCompaniesOrBranchesPage.submit()).click();
+      $(EstimatePage.submit()).click();
+      expect(browser.getUrl()).to.contain(SectionSummaryPage.url());
+    });
+    it("When I add my own item and relevant data, Then after I answer yes on additional items page I should be able to choose an item from the items list and add relevant data about it.", () => {
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.yes()).click();
+      $(AnyCompaniesOrBranchesDrivingQuestionPage.submit()).click();
+      $(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).setValue("Company A");
+      $(AnyCompaniesOrBranchesAddPage.registrationNumber()).setValue("0123456789");
+      $(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).click();
+      $(AnyCompaniesOrBranchesAddPage.submit()).click();
+      $(AnyCompaniesOrBranchesPage.yes()).click();
+      $(AnyCompaniesOrBranchesPage.submit()).click();
+      expect($(AnyCompaniesOrBranchesAddPage.companyOrBranchName()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesAddPage.registrationNumber()).isExisting()).to.be.true;
+      expect($(AnyCompaniesOrBranchesAddPage.authorisedInsurerRadioYes()).isExisting()).to.be.true;
```

Comment:
Should test the value. Existing doesn't prove the value is as expected. Does the correct item have the correct answer. This would pass even if items 2 answers were being used as item 1.

Reply:
Checking for value/correct answer now.